doc/guides/releases.md

```diff
@@ -398,7 +398,19 @@ Create a pull request targeting the correct release line. For example, a
 `v5.3.0-proposal` PR should target `v5.x`, not master. Paste the CHANGELOG
 modifications into the body of the PR so that collaborators can see what is
 changing. These PRs should be left open for at least 24 hours, and can be
-updated as new commits land.
+updated as new commits land. If the CHANGELOG is long enough that it slows
+down the GitHub UI, consider pasting the commits into `<details>` tags or
+in follow up comments.
```

Comment:
What is meant by "follow up comments" ? Is this referring to github comments? If so, that would require someone to view the changelog through github to view all of the commits which doesn't seem like a good idea.

Reply:
Yes, it refers to GitHub comments.

That's correct - they would need to view Changelog in the GitHub conversation. This is only when the Changelog gets long enough that the GitHub API slows down because the pull request description is too large. This has only happened once to me during a release, but the UI made the release conversation impossible.

If the pull request description doesn't load and/or will not update due to the size of the Changelog, then this is a better option than having no Changelog in the pull request at all.

Reply:
@mscdex does this explanation make sense or do you think we should propose an alternative solution?

Reply:
I think we should only suggest using the details tag in that scenario as it will keep the changes in the repo, which is less dependent on the git host, allows the changelog to be viewed offline, and allows us to keep changelogs with consistent availability (e.g. we won't have situations where some changelogs are in the repo and others are not).

Reply:
The intent here is not to change what's in the repo. It's about the copy of the changelog that releasers paste into the release proposal pull request thread to simplify reviews

Reply:
That's correct. I'll make modifications to the docs to make that more clear.